lib/handlers/map.get.js

```diff
@@ -1,35 +1,31 @@
 'use strict';
 
 const HttpOutgoing = require('../classes/http-outgoing');
+const validators = require('../utils/validators');
+const httpError = require('http-errors');
 const ImportMap = require('../classes/import-map');
 
 const handler = async (sink, req, org, name, version) => {
-    if (typeof org !== 'string' || org === '') {
-        throw new TypeError(
-            ':org is a required url parameter and must be a string'
-        );
-    }
-
-    if (typeof name !== 'string' || name === '') {
-        throw new TypeError(
-            ':name is a required url parameter and must be a string'
-        );
-    }
-
-    if (typeof version !== 'string' || version === '') {
-        throw new TypeError(
-            ':version is a required url parameter and must be a string'
-        );
+    try {
+        validators.version(version);
+        validators.name(name);
+        validators.org(org);
+    } catch (error) {
+        throw new httpError(404, 'Not found');
```

Comment:
should this be a blanket 404? Depending on the validation error it could be different 4xx errors?

Reply:
See my comment in: https://github.com/asset-pipe/core/pull/11#discussion_r336848921